### DISCUSSION TITLE
Newsletter module

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#14922] [Q&A] [answered]
Is there a newsletter module for OrchardCore?

**Comment by hishamco** [comment]
This is sort of community-driven module, you can open an issue in OCC https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules, you can start on in this if you want and might help you

**Comment by Piedone** [comment, marked as answer]
What do you imagine a newsletter module doing? Newsletter management is a huge feature set, but you can more easily integrate with a platform like [Mailchimp](https://mailchimp.com/). A simple sign-up link or an Html Widget with their sign-up embed code can be a good first step.

**Reply by DotCat1985** [reply]
I imagine a newsletter module similar to Mailchimp. Since the newsletter management is a huge feature, I could embed code. Thank you.

**Reply by hishamco** [reply]
Or you can start with something simple

**Reply by DotCat1985** [reply]
@hishamco It's not necessary, because I could- easily embed code from an external service, like Mailchimp. :)

**Reply by hishamco** [reply]
Yes, you can, but there will be no integration with other parts in Orchard Core service if you need to
